test/write.test.ts

```diff
@@ -346,4 +346,133 @@ describe("commands/write", function () {
 
     equal(resolverMock.calledOnce, true);
   });
+
+  test("Write works with GRANT statement", async function () {
```

Comment:
I LOVE LOVE LOVE that without fail, @joewagner _always_ including tests in his PRs.

Reply:
This one was mostly tests haha!

Reply:
:boom: